### PR TITLE
demo: update Table demo

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [ ] 📝 Site / documentation improvement
- [x] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | demo: update Table demo |
| 🇨🇳 Chinese | 新增更多列，目前在 2k 分辨率下看不到左右滚动的效果 |

### CODE REVIEW

File: components/table/demo/fixed-columns-header.tsx [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+
+const useStyle = createStyles(({ css, token }) => {
+  const { antCls } = token;
+  return {
+    customTable: css`
+      ${antCls}-table {
+        ${antCls}-table-container {
+          ${antCls}-table-body,
+          ${antCls}-table-content {
+            scrollbar-width: thin;
```
Comment (li-jia-nan) [inline components/table/demo/fixed-columns-header.tsx:14]: 原生的滚动条在 Windows 下太丑，这里用较细的样式

File: components/table/demo/fixed-columns-header.tsx [hunk 2]
```diff
@@ -67,6 +85,18 @@ const columns: TableColumnsType<DataType> = [
     width: 150,
   },
   { title: 'Column 8', dataIndex: 'address', key: '8' },
+  { title: 'Column 9', dataIndex: 'address', key: '9' },
```
Comment (li-jia-nan) [inline components/table/demo/fixed-columns-header.tsx:88]: 2k 分辨率下表格列太少，看不到滚动效果

File: components/table/demo/fixed-columns-header.tsx [hunk 3]
```diff
@@ -76,18 +106,23 @@ const columns: TableColumnsType<DataType> = [
...
+  address: `London, Park Lane no. ${i}`,
+}));
 
-const App: React.FC = () => (
-  <Table columns={columns} dataSource={data} scroll={{ x: 1500, y: 300 }} />
-);
+const App: React.FC = () => {
+  const { styles } = useStyle();
+  return (
+    <Table<DataType>
```
Comment (li-jia-nan) [inline components/table/demo/fixed-columns-header.tsx:119]: 加上 DataType 泛型，有更好的类型提示

File: components/table/demo/fixed-columns-header.tsx [hunk 4]
```diff
@@ -76,18 +106,23 @@ const columns: TableColumnsType<DataType> = [
...
+      className={styles.customTable}
+      columns={columns}
+      dataSource={dataSource}
+      scroll={{ x: 'max-content', y: 55 * 5 }}
```
Comment (li-jia-nan) [inline components/table/demo/fixed-columns-header.tsx:123]: 这里的滚动 y 值改成了每行高度 * 整数倍，可以正好展示完整的 5 行

File: components/table/demo/resizable-column.tsx [hunk 5]
```diff
@@ -44,6 +37,30 @@ const ResizableTitle = (
   );
 };
 
+const data: DataType[] = [
```
Comment (li-jia-nan) [inline components/table/demo/resizable-column.tsx:40]: 数组常量提到组件外面，避免每次 render 都重新声明一遍

**Review by afc163 — APPROVED** [review]

**Pull request merged**